### ISSUE TITLE
Buffs that improve with talent points, buff text displays as if no talents, functions as intended.

### ISSUE THREAD

**Issue by Brunelleschino** [elysium-project/classic-bug-tracker#749]
at lvl 48, power word fortitude increases stamina by 43. The first talent point in the talent above increased it correctly to 49 but the second talent point i spent reduced it to 43 again.

**Comment by Thenari** [comment]
Is it the buff text saying it inncorrect or did you do the math and find you were recieving the wrong amount.

I know my imps blood pact buff. Theres talents that increase it but the buff text does not reflect that but the math in my character sheet do.

Im not at a computer right now.

**Comment by Brunelleschino** [comment]
Tried it agai: only receive 430hp more instead of 559 as should be with 30% more due to talent.

**Comment by Thenari** [comment]
Alrighty ill confirm im coming home with groceries

**Comment by Thenari** [comment]
I feel like its probs just the rank you are using at 48... cause im sure someone would of noticed if the best one in the game was lying... ill check all ranks with no talent, 1point in talent, etc.

**Comment by Brunelleschino** [comment]
Unlearning my talent points and reskilling solved the problem.

**Comment by Thenari** [comment]
Well thats a weird solution... but at least its working i guess

**Comment by Thenari** [comment]
If someone can fix buffs with talent bonuses buff description thats all thats left to this. Otherwise it functions normally just wrong spell text

**Comment by Thenari** [comment]
This is a minor visual glitch that is based on the buff description not changing with talent upgrades its almost not worth fixing but its here if anyone wants to look at it

**Comment by Thenari** [comment]
If you wanna help let us know a list of buffs that say "+ x" but actually "+ y"

**Comment by Thenari** [comment]
"Hi,

Nighthaven pvp 1.9

As a warrior I've got 5 talent point in battle shout (+25%). If i hover over the spell it tells me +231 attack power. However, when i cast the buff and i hover over the top right. It tells me +192 attack power.

Kind regards
Sander (Mysense)"

**Comment by mikeomaco** [comment]
Hello, hope you are well.
Improved Imp still has a graphics bug (3864 hp without bloodpact.  With bloodpact 4484)
The buff bar icon and spell book values do not match.

These were taken during the Patch 1.10.
These were taken after a WDB clear.
Hope this helps :)

![wowscrnshot_101318_022024](https://user-images.githubusercontent.com/44105383/46900182-0808a280-ce96-11e8-9535-766455f362f1.png)
![wowscrnshot_101318_022019](https://user-images.githubusercontent.com/44105383/46900183-0808a280-ce96-11e8-8956-52bab5a8efd0.png)

**Comment by Derathul** [comment]
Hey I have the same issue. I have the talent "Improved Power Word: Fortitude" that should increase my Stamina buff by 30% and although it shows in my spell book as "Increases stamina by 41" when I have it on in the upper right it says "Increased by 32". In my character tab I checked and it increases by correct amount but it's confusing during parties/instances with more than one priest. For example yesterday I joined a group and there was a priest already that buffed other party members but when I got to them and buffed them it appeared my buff was a lot stronger but I couldn't figure out who has "mine" buff and who has "the other's guy" buff. :)
Hope it helps :)
